src/app/PCM/templates/tpl-detail-view/tpl-application-detail-view/checkLists.js

```diff
@@ -8,23 +8,27 @@ import {
 
 import Ask from 'app/ASKS/ask'
 
-const CheckLists = ({ code: passedQuestionCode }) => {
+const CheckLists = ({ code: passedQuestionCode, isStudent }) => {
   const CheckListItem = ({ parentCode, questionCode }) => {
-    return <Ask parentCode={parentCode} questionCode={questionCode} />
+    return <Ask overrideComponent={'approve'} parentCode={parentCode} questionCode={questionCode} />
   }
 
   return (
     <Box>
-      <Grid gap="1rem">
+      <Grid>
         <CheckListItem parentCode={passedQuestionCode} questionCode={matchesVisaQuestionCode} />
-        <CheckListItem
-          parentCode={passedQuestionCode}
-          questionCode={matchesEmpContractQuestionCode}
-        />
-        <CheckListItem
-          parentCode={passedQuestionCode}
-          questionCode={websiteInvestigatedQuestionCode}
-        />
+        {!isStudent && (
```

Comment:
Create an object with different questioncode and use a mapper to display the CheckListItem rather than writing it twice and checking for if it is a student or not.

Reply:
Now uses a mapping function